### PR TITLE
[CDAP-5101] [CDAP-5189] [CDAP-5188] [CDAP-5148] Tracker Iterations 1

### PR DESCRIPTION
**Pull request by elfenheart**
- Add query parameters for lineage
- Remove schema from system properties
- Ability to click open an element in new tab
- Query found in -> Search term found in
- Lineage multiple methods (duplicate or not) as query params to lineage

### CODE REVIEW

File: cdap-ui/app/features/tracker/directives/lineage/lineage.js [hunk 1]
```diff
@@ -58,18 +58,22 @@ function LineageController ($scope, jsPlumb, $timeout, $state, LineageStore, myT
         lineWidth: 2,
         strokeStyle: 'rgba(0,0,0, 1)'
       },
-      Connector: [ 'Flowchart', {gap: 0, stub: [10, 15], alwaysRespectStubs: true, cornerRadius: 3} ],
-      Endpoints: ['Blank', 'Blank']
+      // Connector: [ 'Flowchart', {gap: 0, stub: [10, 15], alwaysRespectStubs: true, cornerRadius: 3} ],
```
Comment (anwar6953) [inline cdap-ui/app/features/tracker/directives/lineage/lineage.js]: remove?

File: cdap-ui/app/features/tracker/directives/lineage/lineage.js [hunk 2]
```diff
@@ -58,18 +58,22 @@ function LineageController ($scope, jsPlumb, $timeout, $state, LineageStore, myT
...
   });
 
-  vm.nodeClick = (node) => {
+  vm.nodeClick = (event, node) => {
     let nodeInfo = vm.uniqueNodes[node.uniqueNodeId];
     if (nodeInfo.nodeType === 'data') {
-      $state.go('tracker.entity.metadata', { entityType: nodeInfo.entityType, entityId: nodeInfo.entityId });
+      return;
     } else {
+      event.preventDefault(); // prevent JS error on nonexistant state
```
Comment (anwar6953) [inline cdap-ui/app/features/tracker/directives/lineage/lineage.js]: nit - nonexistent

Comment (astewes) [inline cdap-ui/app/features/tracker/directives/lineage/lineage.js]: not sure if this justifies a commit, but "nonexistant" is misspelled

File: cdap-ui/app/features/tracker/directives/lineage/lineage.js (hunk 1 above)
Comment (elfenheart) [inline cdap-ui/app/features/tracker/directives/lineage/lineage.js]: Not yet. Might need to revert back to this depending on what we decide with lineage. Lineage will go through couple more iterations

**Comment by astewes** [comment]
other than the minor comments already called out :+1: LGTM

**Pull request merged**